iceoryx_dds/include/iceoryx_dds/internal/gateway/dds_to_iox.inl

```diff
@@ -42,7 +42,7 @@ inline void DDS2IceoryxGateway<channel_t, gateway_t>::loadConfiguration(const co
             LogDebug() << "[DDS2IceoryxGateway] Setting up channel for service: {"
                        << serviceDescription.getServiceIDString() << ", " << serviceDescription.getInstanceIDString()
                        << ", " << serviceDescription.getEventIDString() << "}";
-            setupChannel(serviceDescription);
+            setupChannel(serviceDescription, popo::PublisherOptions());
```

Comment:
So currently in the DDS gateway the history is always 0?

Reply:
yes, I wanted to keep the behavior the same. Shall we change it?